snapcraft/plugins/python2.py

```diff
@@ -40,182 +40,35 @@
       A list of dependencies to get from PyPi
 """
 
-import glob
 import logging
 import os
-import re
 
-import snapcraft
-from snapcraft import common
+from snapcraft.plugins import python3
 
 logger = logging.getLogger(__name__)
 
 
-class Python2Plugin(snapcraft.BasePlugin):
+class Python2Plugin(python3.Python3Plugin):
```

Comment:
and where would `BasePythonPlugin` live? in the `python3` module or the `python2` one?